evaluator/autoware_planning_evaluator/src/metrics_calculator.cpp

```diff
@@ -20,6 +20,7 @@
 #include "autoware/planning_evaluator/metrics/stability_metrics.hpp"
 #include "autoware/planning_evaluator/metrics/trajectory_metrics.hpp"
 #include "autoware/universe_utils/geometry/geometry.hpp"
+
```

Comment:
remove diff

Reply:
adressed in 2f06b233d799a6918f6c036ace480f7f87b07c76